.circleci/config.yml

```diff
@@ -41,28 +26,28 @@ jobs:
           paths:
             - node_modules
           key: v2-dependencies-npm-{{ checksum "package-lock.json" }}
-      - run:
-          name: Check Ruby style
-          command: bundle exec standardrb
-      - run:
-          name: Check JavaScript style
-          command: npx standard
-      - run: npm run uswds-build
+      # - run:
+      #     name: Check JavaScript style
+      #     command: npx standard
+      - build: npm run build
```

Comment:
suggestion (non-blocking, we can do it another time): Running `lint` before `build` would fail faster, and `lint` doesn't depend on `build`.